### ISSUE TITLE
ARM (not ARM64) image support

### ISSUE THREAD

**Issue by cerebrate** [kubernetes-sigs/descheduler#475]
**Is your feature request related to a problem? Please describe.**
I have a Raspberry Pi-based (i.e. arm) cluster I would like to be able to use descheduler on.

**Describe the solution you'd like**
I would like arm supported to be added to the existing multi-arch build. Since building an arm image for descheduler is trivial, this should be easy to add.

**Describe alternatives you've considered**
I could just continue building my own multi-arch version of descheduler, but it seems wasteful when I'm almost certainly not the only one wanting to use it on arm.

**What version of descheduler are you using?**

descheduler version: 0.20.0

**Comment by seanmalloy** [comment]
@cerebrate so you would like an official descheduler container image for ARMv7?

```
GOOS=linux GOARCH=arm GOARM=7 go build ...
```

As you mentioned this should be fairly easy to add, so please confirm that you need ARMv7. We can try to get this setup for the descheduler v0.21.0 release. Thanks!

**Comment by cerebrate** [comment]
Yep, that's exactly the one I'd like. Thanks!

**Comment by seanmalloy** [comment]
@damemi and @ingvagabund any concerns with adding one more container image? We would end up with three(AMD64, ARM64, and ARMv7) if we implement this.

/assign

**Comment by damemi** [comment]
@seanmalloy I don't have any problem with that, feel free to open the PRs :)

**Comment by ingvagabund** [comment]
No problem here either :)

**Comment by seanmalloy** [comment]
The ARMv7 image will be available with the descheduler v0.21.0 release.

You can pull the below image to test it prior to the release if you would like. Do **NOT** use the below image in production.
```
docker pull gcr.io/k8s-staging-descheduler/descheduler:v20210310-descheduler-helm-chart-0.20.0-64-g0a11b5a13
```

**Comment by cerebrate** [comment]
Works great for me in testing, thanks!
